package.json

```diff
@@ -15,6 +15,7 @@
     "rules"
   ],
   "scripts": {
+    "eslint-check": "eslint --print-config . | eslint-config-prettier-check",
```

Comment:
Do you want to add this into the `test` npm script?

Reply:
i think i will in a different pr. there are conflicting rules right now and it's not bringing in all the rules